### PR TITLE
chore: bump reth

### PR DESCRIPTION
**Pull request by mattsse**
recent breaking api changes

also includes sequencer url fixes

### CODE REVIEW

File: crates/node/src/node.rs [hunk 1]
```diff
@@ -104,7 +107,7 @@ pub struct AlphaNetExecutorBuilder;
 
 impl<Node> ExecutorBuilder<Node> for AlphaNetExecutorBuilder
 where
-    Node: FullNodeTypes<Types: NodeTypes<ChainSpec = ChainSpec>>,
+    Node: FullNodeTypes<Types: NodeTypes<ChainSpec = OpChainSpec>>,
```
Comment (onbjerg) [inline crates/node/src/node.rs:110]: why this change to opchainspec? last i checked it didnt do anything and was just a super shallow wrapper that we cast down to `ChainSpec` anyway

Comment (mattsse) [reply crates/node/src/node.rs:110]: there's more stuff wip, because this now allows us to change the OpChainSpec internals, like no dyn Hardforks

Comment (onbjerg) [reply crates/node/src/node.rs:110]: gotcha

**Review by onbjerg — APPROVED** [review]
lgtm once ci passes
